misk-grpc-vanilla/src/main/kotlin/misk/grpc/vanilla/wireserver/RouteGuideWireGrpc.kt

```diff
@@ -0,0 +1,323 @@
+package misk.grpc.vanilla.wireserver
```

Comment:
This entire file should be generated. We can either build this into (or fork) the Wire compiler, fork the Google Java compiler, or build it as a `protoc` plugin

Reply:
I think this is the key piece

Reply:
It is! Defining what should get generated is step 1. Generating it should be fairly easy.

Would you like to try doing it? I'm happy to get it started.